packages/icons-business-suite/package.json

```diff
@@ -26,7 +26,8 @@
     "directory": "packages/icons-business-suite"
   },
   "dependencies": {
-    "@ui5/webcomponents-base": "1.9.1"
+    "@ui5/webcomponents-base": "1.9.1",
+    "typescript": "^4.9.3"
```

Comment:
Why do we need typescript as a dependency? In my opinion it should be a dev dependency. Same applies for the other packages.

Reply:
We will change that, thank you @MarcusNotheis